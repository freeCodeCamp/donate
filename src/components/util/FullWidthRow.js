import React from 'react';
import PropTypes from 'prop-types';
import { Row, Col } from 'react-bootstrap';

function FullWidthRow({ children }) {
  return (
    <Row>
      <Col sm={8} smOffset={2} xs={12}>
        {children}
      </Col>
    </Row>
  );
}

FullWidthRow.displayName = 'FullWidthRow';
FullWidthRow.propTypes = {
  children: PropTypes.any
};

export default FullWidthRow;
