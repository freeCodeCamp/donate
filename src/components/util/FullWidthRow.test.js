/* global expect */

import React from 'react';
import renderer from 'react-test-renderer';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import FullWidthRow from './FullWidthRow';

Enzyme.configure({ adapter: new Adapter() });

test('<FullWidthRow /> snapshot', () => {
  const component = renderer.create(<FullWidthRow />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
