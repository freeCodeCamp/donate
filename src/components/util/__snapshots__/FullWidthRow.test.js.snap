// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FullWidthRow /> snapshot 1`] = `
<div
  className="row"
>
  <div
    className="col-sm-8 col-sm-offset-2 col-xs-12"
  />
</div>
`;
